guides/user/pipeline/managing-pipelines/index.md

```diff
@@ -85,14 +85,16 @@ pipeline.
 
 > **Warning**: the JSON you write here is not validated -- Edit as JSON
 essentially allows you to modify the pipeline via a free-form textbox. **Be
-careful!** It is very easy to break the pipeline.
+careful!** It is very easy to break the pipeline, although you can restore from
```

Comment:
s/It is/It's/
https://developers.google.com/style/contractions

Reply:
And feel free to change all such instances throughout. But don't lose any sleep over it.